crates/cargo-contract/src/cmd/extrinsics/integration_tests.rs

```diff
@@ -71,11 +71,6 @@ impl ContractsNodeProcess {
         let mut attempts = 1;
         let client = loop {
             thread::sleep(time::Duration::from_secs(1));
-            tracing::debug!(
-                "Connecting to contracts enabled node, attempt {}/{}",
-                attempts,
-                MAX_ATTEMPTS
-            );
```

Comment:
As this one is in a helper function and not directly in the test, let's bring it back.